universal-application-tool-0.0.1/app/modules/SecurityModule.java

```diff
@@ -122,10 +113,11 @@ protected Config provideConfig(GuestClient guestClient, FormClient formClient) {
                 forbidden("403 forbidden").as(HttpConstants.HTML_CONTENT_TYPE)));
     Config config = new Config();
     config.setClients(clients);
-    config.addAuthorizer("uatadmin", new RequireAllRolesAuthorizer("ROLE_UAT_ADMIN"));
-    config.addAuthorizer("programadmin", new RequireAllRolesAuthorizer("ROLE_PROGRAM_ADMIN"));
-    config.addAuthorizer("applicant", new RequireAllRolesAuthorizer("ROLE_APPLICANT"));
-    config.addAuthorizer("intermediary", new RequireAllRolesAuthorizer("ROLE_TI"));
+    config.addAuthorizer(
+        "uatadmin", new RequireAllRolesAuthorizer(Roles.ROLE_UAT_ADMIN.toString()));
```

Comment:
issue (blocking): I think I was totally wrong about not needing a program admin role for MVP (and I can't comment on the old thread apparently). Sorry about that! I realize we do need the program admin to be able to download the applications from UAT.